kdump/include/runtest.sh

```diff
@@ -370,7 +370,7 @@ PrepareKdump()
             # Kdump service will not be enabled if crashkernel=auto && system
             # memory is less the threshold required by kdump service.
             /bin/systemctl enable kdump.service || /sbin/chkconfig kdump on
-
+            rpm -q grubby || InstallPackages grubby
```

Comment:
Maybe better this way, so we don't add extra output? Or do you want the output to be there, to aid debugging?
```suggestion
            rpm --quiet -q grubby || InstallPackages grubby
```